GithubIssueTagger/InsertionChangelogGenerator.cs

```diff
@@ -0,0 +1,212 @@
+﻿using Octokit;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace GithubIssueTagger
+{
+    public class InsertionChangelogGenerator
+    {
+        public static async Task GenerateInsertionChangelog(GitHubClient gitHubClient)
+        {
+            // Adjust the sha/repo
+            var startSha = "58795a41a3c8c9d63801fb5072a2e68e48d7507e";
```

Comment:
Could you please make this a command-line parameter? I really don't want to have to recompile this tool just to change a hash.

Reply:
I can change it. 
Now this is part of the GitHubIssueTagger tool, do we want the default for this tool to be this or a separate tool?

Reply:
@NuGet/nuget-client 

What do we think here? 
Same tool, make the changelog generator the default?
New tool?
Require recompiling? 

fwiw, this is a minimal amount of code, so the recompilation really won't be a big deal.

Reply:
Is change log generation a different "mode" to tagging? similar to how `dotnet build` is different to `dotnet add`?

I've never used the issue tagger before, so I wouldn't want to accidentally tag something when I intended to generate the commit list for an insertion, so if they're not separated, then I'd certainly want them to be.

As for requiring recompilation or command line argument, I think a command line argument is definately preferable, but this is a non-shipping tool. I'd just merge whatever is convient now, and whoever wants to make it command line argument can modify the code.  I don't even create pull requests for my CI analysis tools, I just push straight to main (should I stop, and start creating PRs?)

Reply:
> (should I stop, and start creating PRs?)

I vote no. It ain't called entropy for nothing!

Reply:
I changed it to a separate project and a global tool.